astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -31,6 +38,23 @@
 from astronomer.providers.utils.typing_compat import Context
 
 
+def _check_queries_finish(conn: SnowflakeConnection, query_ids: list[str]) -> bool:
```

Comment:
doc-string in this class would be useful

Reply:
Sure!